### PR TITLE
Specify the Correct Data Type

### PR DESCRIPTION
**Pull request by evanplaice**
- basics.network.url -> uri
- publications.url -> uri
- meta.canonical -> uri

Ref: #320

### CODE REVIEW

**Comment by evanplaice** [comment]
@thomasdavis I need a review on this as well since it affects the schema.

File: schema.json [hunk 1]
```diff
@@ -71,7 +71,8 @@
             "properties": {
               "network": {
                 "type": "string",
-                "description": "e.g. Facebook or Twitter"
+                "description": "e.g. Facebook or Twitter",
+                "format": "uri"
```
Comment (mathieudutour) [inline schema.json]: shouldn't this be in the `url` property instead of the `network`?

Comment (stp-ip) [reply schema.json]: Second that.

Comment (evanplaice) [reply schema.json]: Yes. My mistake.

**Comment by evanplaice** [comment]
@stp-ip Quick question. Who has publish access to the JSONResume org on NPM?

**Comment by stp-ip** [comment]
@thomasdavis is the only one with access to npm, website, domain, registry and anything else that isn't on github.

**Comment by thomasdavis** [comment]
lgtm 

I will see how I can get you npm publish rights.

**Comment by thomasdavis** [comment]
@evanplaice I will need your npm username

**Comment by thomasdavis** [comment]
Screen Shot 2020-04-21 at 9 48 59 am

**Comment by evanplaice** [comment]
@thomasdavis Thanks. My NPM username is `evanplaice`. What can I say, I'm like to keep an air of mystery.

With the secret applied here, the CD publishing workflow should work even without NPM access.

NPM access will be handy when I move on to work on the other repos (ex resume-cli) as those will need secret tokens added as well.

**Comment by thomasdavis** [comment]
@evanplaice I've also added the token to the resume-cli repo

**Comment by evanplaice** [comment]
Both CI and CD publish are working 🎉🎉🎉🎉

**Comment by thomasdavis** [comment]
@evanplaice Yeah the CD is beautiful, great job

**Pull request merged**